commands/account/createOverride.ts

```diff
@@ -0,0 +1,75 @@
+import fs from 'fs-extra';
+import path from 'path';
+import { Argv, ArgumentsCamelCase } from 'yargs';
+import { getCwd } from '@hubspot/local-dev-lib/path';
+import { logger } from '@hubspot/local-dev-lib/logger';
+import { getConfigPath, getAccountId } from '@hubspot/local-dev-lib/config';
+import { addConfigOptions } from '../../lib/commonOpts';
+import { i18n } from '../../lib/lang';
+import { EXIT_CODES } from '../../lib/enums/exitCodes';
+import { selectAccountFromConfig } from '../../lib/prompts/accountsPrompt';
+import { CommonArgs, ConfigArgs } from '../../types/Yargs';
+
+const i18nKey = 'commands.account.subcommands.createOverride';
+
+export const describe = null; // i18n(`${i18nKey}.describe`);
+
+export const command = 'create-override [account]';
+
+type AccountInfoArgs = CommonArgs &
+  ConfigArgs & {
+    account?: string | number;
+  };
+
+export async function handler(
+  args: ArgumentsCamelCase<AccountInfoArgs>
+): Promise<void> {
+  let overrideDefaultAccount = args.account;
+
+  if (!overrideDefaultAccount) {
+    overrideDefaultAccount = await selectAccountFromConfig();
+  } else if (
+    (typeof overrideDefaultAccount !== 'string' &&
+      typeof overrideDefaultAccount !== 'number') ||
+    !getAccountId(overrideDefaultAccount)
```

Comment:
This can be simplified to just `!getAccountId()` since yargs will yell at the user if the input type of the positional doesn't match what we're expecting

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1349/commits/5b758d2e4b814cb8abc89bba8cb54cb3c25df7b1